Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
+use EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SystemInfoDumpCommand extends ContainerAwareCommand
 {
     /**
+     * System info collector registry.
+     *
+     * @var \EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry
+     */
+    private $registry;
+
+    public function __construct(SystemInfoCollectorRegistry $registry)
+    {
+        $this->registry = $registry;
+
+        parent::__construct();
+    }
+
+    /**
      * Define command and input options.
      */
     protected function configure()
     {
         $this
             ->setName('ez-support-tools:dump-info')
-            ->setDescription('Collects system information and dumps it.')
+            ->setDescription(<<<'EOD'
```

Comment:
Thanks, that's useful. Neither console tutorials nor classdoc mention it afaict.